src/cloud-api-adaptor/podvm/Makefile

```diff
@@ -84,7 +84,7 @@ $(IMAGE_FILE): $(BINARIES) $(FILES) setopts
 		qemu-img create -f qcow2 "se-${IMAGE_FILE}" 100G; \
 	fi
 	if [ "${ARCH}" = "x86_64" ]; then \
-		packer init qcow2/rhel/qemu-rhel.pkr.hcl; \
+		packer plugins install github.com/hashicorp/qemu v1.1.0; \
```

Comment:
As we are skipping the init of `qcow2/rhel/qemu-rhel.pkr.hcl` do we still need that file at all? As there are other things rather than just the plugin installs I wonder if it's safer to do the required_plugins removal in there, but still call `packer init qcow2/rhel/qemu-rhel.pkr.hcl;`?

As a disclaimer I have very little packer knowledge, so sorry about that.

Reply:
yes Steve, We still require the qcow2/rhel/qemu-rhel.pkr.hcl file. Despite the successful execution of the Packer build without the packer init command, I've included it as a precautionary measure based on your recommendation.

Thank you.

Reply:
@Saripalli-lavanya few questions
1. Will having required_plugins section be more flexible in case we want to add some additional filtering going forward?
2. Why not just call `packer init qcow2/$(PODVM_DISTRO)` (with required_plugins set properly for all distros) to install the related plugins for every distro since this makefile is generic for all distro?
3. How are you building (from src) and installing the qemu plugin for s390x? I don't see it in the code. Did I miss something ?

Reply:
Hi @bpradipt, Thank you for the review comments, 

1. Qemu plugin for s390x is not available (https://github.com/hashicorp/packer-plugin-qemu/releases) and so we will not be able to install it through required_plugins section or could you please suggest if there is a way to run required plugins only for x86?
2. Sure, updated code to use packer init ./qcow2/$(PODVM_DISTRO).
3. Earlier I had added code to build plugin for s390x from source code but later It seems like for s390x we don't need to build Qemu plugin separately(may be because we are building packer from source code in Dockerfile [here](https://github.com/confidential-containers/cloud-api-adaptor/blob/a5e37f471e0bca98f0ac25b84d0666b6687be5af/src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel#L44)) and so i have removed that code and instead installing package separately for x86. Could you please suggest further here thank you

Reply:
Ok, so if I understand correctly you want a way to install s390x qemu plugin separately as part of build ?
In that case required_plugins section won't work. Can you add a description in the commit message why the required_plugins is being removed. This will help going forward for anyone looking at the changes.

Reply:
Sure, Thank you. I have added description as suggested.